packages/nimble-components/src/rich-text/specs/README.md

```diff
@@ -473,6 +983,47 @@ _Keyboard navigation with toolbar buttons focused_
 
 _Note_: All other keyboard interaction determined by the slotted element will not be defined in this document.
 
+_Keyboard interactions for `@mention`_
+
+| Key                                      | Behavior                                                                                                                            |
+| ---------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| `@`                                      | To open the list of options with the users list                                                                                     |
+|                                          | ----Below are the keyboard interactions when popup is opened----                                                                    |
```

Comment:
In Teams and GitHub I'm seeing that the dropdown is hidden if I use arrow keys to move the text cursor away from the `@` or the partial name next to it. I think we should include this interaction too. I've seen other implementations that keep the dropdown visible for too long and it's annoying to have to press ESC to remove it.

Reply:
I see that you include this in your prototype (nice!) so it just needs to be called out in the spec.